db/migrate/20151025165759_create_user_points.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUserPoints < ActiveRecord::Migration
+  def change
+    create_table :user_points do |t|
+      t.string :reason
+      t.integer :points
+      t.string :operation
+      t.integer :user_id
```

Comment:
@junwen29 @yeshasimaria Where's e belongs to user model? Need associations btw user points and user